.travis.yml

```diff
@@ -125,7 +128,9 @@ install:
 script:
  - getconf _NPROCESSORS_ONLN
  - ${MAKE} -j$(getconf _NPROCESSORS_ONLN) check VERBOSE=1 V=${ENV_VERBOSE}
- - ant build test
+ - if [ -e "/usr/lib/jvm/java-8-oracle/" ]; then export JAVA_HOME=/usr/lib/jvm/java-8-oracle/; fi
+ - mvn -version
```

Comment:
nit: Remove this line. The `mvn -version` at before_install should be the one that counts.